.github/workflows/ci.yml

```diff
@@ -481,6 +486,9 @@ jobs:
             packages/qwik-labs/vite/
             packages/qwik-labs/package.json
           if-no-files-found: error
+      - run: pnpx pkg-pr-new publish --compact ./packages/qwik/dist ./packages/qwik-city/lib dist-dev-eslint-plugin-qwik dist-dev-create-qwik
```

Comment:
`pnpx` is [deprecated](https://github.com/pnpm/pnpm/pull/3652) 
Can we use the new notation pls?

Reply:
yes

Reply:
we can use `pnpm dlx pkg-pr-new@0.0.5` 
we can remove the dep if we will use the `@` notation

Reply:
done

Reply:
OK I figured it out